site/en/docs/extensions/mv3/service_workers/service-worker-lifecycle/index.md

```diff
@@ -66,6 +64,22 @@ Any global variables you set will be lost if the service worker shuts down. Inst
 [CacheStorage API](https://developer.mozilla.org/docs/Web/API/CacheStorage)
 : A persistent storage mechanism for Request and Response object pairs. This API was designed specifically for web service workers and is used to retrieve data from an endpoint. There are a variety of ways to use this API depending on whether and how critical it is that users see up-to-date data. For more information, see [The Offline Cookbook](​​https://web.dev/offline-cookbook). Unless you're specifically proxying network requests via the fetch handler, you should use `chrome.storage`.
 
-## Be careful with timeouts {: #timeouts }
+### Keeping service workers alive
```

Comment:
When did Chrome start resetting the timer on extension API calls?

Reply:
That's in 110. I tweaked the language to make that clearer.